.github/workflows/continuous-integration.yml

```diff
@@ -24,12 +24,21 @@ jobs:
         run: bash scripts/ci/inspect_binary_file_sizes.sh ${{ github.sha }} ${{ github.sha }}
 
       - name: set up Node.js
+        id: setup-node
         uses: actions/setup-node@v3
         with:
           cache: npm
```

Comment:
if node_modules cache works fine then this can be removed
(wasn't working anyway but I'm still not sure why...)

Reply:
impressively, it changes nothing, will remove